README.md

```diff
@@ -56,19 +69,26 @@ The SDO owner services are packaged as a single docker container that can be run
 
 **On a Horizon "admin" host** run these simple SDO APIs to verify that the services within the docker container are accessible and responding properly. (A Horizon admin host is one that has the `horizon-cli` package, which provides the `hzn` command, and has the environment variables `HZN_EXCHANGE_URL`, `HZN_SDO_SVC_URL`, `HZN_ORG_ID`, and `HZN_EXCHANGE_USER_AUTH` set correctly for your Horizon management hub.)
 
-1. Query the OCS API version:
+1. Export your environment variables for `HZN_SDO_SVC_URL` and `SDO_RV_URL` for the subsequent steps.
+
+   ```bash
+   export HZN_SDO_SVC_URL=http://<SDO_OWNER_SVC_HOST>:9008/api
+   export SDO_RV_URL=http://<SDO_OWNER_SVC_HOST>:8040
```

Comment:
Also should repeat the export of HZN_ORG_ID and HZN_EXCHANGE_USER_AUTH here, because this is likely a different machine.